modules/acquia_cms_site_studio/composer.json

```diff
@@ -8,6 +8,7 @@
         "acquia/cohesion-theme": "~7.4.0",
         "drupal/acquia_cms_common": "1.x-dev || 2.x-dev || 3.x-dev",
         "drupal/collapsiblock": "^4.0",
+        "drupal/sitestudio_gin": "^1",
```

Comment:
@rajeshreeputra I don't think we should add this dependency. Let's remove this.
cc: @vipin-mittal-acquia

Reply:
@vishalkhode1 Is there a particular reason not to include this functionality by default?